docs/deploy/aws.md

```diff
@@ -60,14 +92,30 @@ If you already have a database, take note of the connection string and skip to t
    **Note:** It's preferable to run the binary as a service using systemd or your operating system's facilities.
 
 ### On ECS
-To support container-based environments like AWS ECS, lakeFS can be configured using environment variables. Here is a `docker run` 
-command to demonstrate starting lakeFS using Docker:
-
+To support container-based environments like AWS ECS, lakeFS can be configured using environment variables. Here are a couple of `docker run` 
+commands to demonstrate starting lakeFS using Docker:
+#### With PostgreSQL
+```sh
+docker run \
+  --name lakefs \
+  -p 8000:8000 \
+  -e LAKEFS_DATABASE_KV_ENABLED="true"
+  -e LAKEFS_DATABASE_TYPE: "dynamodb" \
```

Comment:
Where's the table name? That one is actually important.

Reply:
Added.
Although, we do have a default table name, which is workable